plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlDistributedQueries.java

```diff
@@ -120,6 +120,11 @@ protected TestTable createTableWithDefaultColumns()
             return Optional.empty();
         }
 
+        if (typeName.contains("char(1)")) {
```

Comment:
we use `char(1)` to denote "the `char` test case covering case insensitiveness".
That's very indirect way of saying so. Make `DataMappingTestSetup`'s literals part of the condition.